src/Driver/PgSQL/Exception/UnknownParameter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Doctrine\DBAL\Driver\PgSQL\Exception;
+
+use Doctrine\DBAL\Driver\AbstractException;
+
+use function sprintf;
+
+/** @psalm-immutable */
+final class UnknownParameter extends AbstractException
+{
+    public static function new(string $param): self
+    {
+        return new self(
+            sprintf('Could not find parameter %d in the SQL statement', $param),
```

Comment:
Should it be `%s` here?

```suggestion
            sprintf('Could not find parameter %s in the SQL statement', $param),
```

Reply:
Yes